app/src/androidTest/java/io/novafoundation/nova/MoonbaseSendIntagrationTest.kt

```diff
@@ -50,13 +55,37 @@ class MoonbaseSendIntagrationTest {
         return EthereumKeypairFactory.generate(seed, junctions)
     }
 
+    private inner class TestSigner(
+        private val testPayload: (SignerPayloadExtrinsic) -> Unit
```

Comment:
As I see we don't use testPayload from outside, so we can remove this redundant variable